mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/JPAAttachment.java

```diff
@@ -0,0 +1,205 @@
+
+    public JPAAttachment(){}
+
+    public JPAAttachment(String attachmentId, byte[] content, String type, Collection<String> owners, Collection<String> messageIds, String cid, Boolean inline){
+        this.attachmentId = attachmentId;
+        this.content = content;
+        this.type = type;
+        this.owners = owners;
+        this.messageIds = messageIds;
+        this.cidValue = cid;
+        this.inline = inline;
+    }
+
+    public String getAttachmentId() {
+        return attachmentId;
+    }
+
+    public void setAttachmentId(String attachmentId) {
+        this.attachmentId = attachmentId;
+    }
+
+    public byte[] getContent() {
+        return content;
+    }
+
+    public void setContent(byte[] content) {
+        this.content = content;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public Collection<String> getOwners() {
+        return owners;
+    }
+
+    public void setOwners(Collection<String> owners) {
+        this.owners = owners;
+    }
+
+    public Collection<String> getMessageIds() {
+        return messageIds;
+    }
+
+    public void setMessageIds(Collection<String> messageId) {
+        this.messageIds = messageId;
+    }
+
+    public String getCidValue() {
+        return cidValue;
+    }
+
+    public void setCidValue(String cidValue) {
+        this.cidValue = cidValue;
+    }
+
+    public Boolean getInline() {
+        return inline;
+    }
+
+    public void setInline(Boolean inline) {
+        this.inline = inline;
+    }
+
+
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {return true;}
+        if (o == null || getClass() != o.getClass()) {return false;}
+        JPAAttachment that = (JPAAttachment) o;
+        return Objects.equals(attachmentId, that.attachmentId)
+                && Arrays.equals(content, that.content)
+                && Objects.equals(type, that.type)
+                && this.owners.containsAll(that.owners)
+                && that.owners.containsAll(this.owners)
+                && this.messageIds.containsAll(that.messageIds)
+                && that.messageIds.containsAll(this.messageIds)
+                && Objects.equals(cidValue, that.cidValue)
+                && Objects.equals(inline, that.inline);
+    }
+
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(this.attachmentId, this.content, this.type, this.owners, this.messageIds, this.cidValue, this.inline);
+    }
+
+    @Override
+    public String toString() {
+        return ToStringBuilder.reflectionToString(this);
+    }
+
+    public Attachment toAttachment() {
+        return Attachment.builder()
+                .attachmentId(AttachmentId.from(this.attachmentId))
+                .bytes(this.content)
+                .type(this.type)
+                .build();
+
+    }
+
+    public static JPAAttachment from(Attachment attachment) {
```

Comment:
could you move static methods to the top of the class? We are trying to apply the same convention in James